test/functional/test_framework/util.py

```diff
@@ -37,6 +37,10 @@ def assert_equal(thing1, thing2, *args):
     if thing1 != thing2 or any(thing1 != arg for arg in args):
         raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
 
+def assert_not_equal(thing1, thing2):
```

Comment:
You've added this but not used it. If it's not being used, can you remove it from this PR?